x/foundation/README.md

```diff
@@ -598,6 +610,14 @@ the treasury.
 | to            | {toAddress}     |
 | amount        | {amount}        |
 
+## EventGovMint
+
+`EventGovMint` is an event emitted when coins are minted by proposal
```

Comment:
Please mind the period. Also, other events does not state about the triggering proposal, because it is evident from the concept.